dashboard/src/components/Config/AppRepoList/AppRepoRefreshAllButton.tsx

```diff
@@ -0,0 +1,35 @@
+import * as React from "react";
+
+import { IAppRepository } from "shared/types";
+import "./AppRepo.css";
+
+interface IAppRepoRefreshAllButtonProps {
+  resyncAllRepos: (names: string[]) => void;
+  repos: IAppRepository[];
+  kubeappsNamespace: string;
+}
+
+export class AppRepoRefreshAllButton extends React.Component<IAppRepoRefreshAllButtonProps> {
+  public render() {
+    return (
+      <div>
+        <button
+          className="button button-primary margin-l-big"
+          onClick={this.handleResyncAllClick()}
+          title="Refresh All App Repositories"
+        >
+          Refresh All
+        </button>
+      </div>
+    );
+  }
+
+  private handleResyncAllClick() {
+    return () => {
```

Comment:
you should be able to use `onClick={this.handleResyncAllClik}` so you don't need to return a function here.

Reply:
Done